src/main.cpp

```diff
@@ -9,6 +9,142 @@
 
 namespace fs = filesystem;
 
+void ArgConfig( args::ArgumentParser &parser) {
+
+    Config config;
+
+    args::ValueFlag<int> columnlimit(parser, "column limit", "Column limit of one line", {"column-limit"});
+    args::ValueFlag<int> indentwidth(parser, "indentation width", "Number of spaces used for indentation", {"indent-width"});
+    args::ValueFlag<int> tabwidth(parser, "tab width", "Number of spaces used per tab", {"tab-width"});
+    args::ValueFlag<int> continuationindentwidth(parser, "Continuation indentation width", "Indent width for continuations line", {"continuation-indent-width"});
+    args::ValueFlag<int> spacesbeforecall(parser, "spaces before call", "Space on function calls", {"spaces-before-call"});
+    args::ValueFlag<int> columntablelimit(parser, "column table limit", "Column limit of each line of a table", {"column-table-limit"});
+    args::ValueFlag<char> tablesep(parser, "table separator", "Character to separate table fields", {"table-sep"});
+    if (tablesep) { config.ArgmapChar["table_sep"] = args::get(tablesep); }
```

Comment:
These checks are wrong and will have no effect here, you gotta do those after we call `parser.ParseCLI(argc, argv);`.
So split the function into 2.
One that setups the arguments.
Another that reads the arguments after the library parsed them.